### ISSUE TITLE
TypeError: Cannot read property 'typeOrTypeDef' of undefined

### ISSUE THREAD

**Issue by glennsyang** [styxlab/gatsby-theme-try-ghost#113]
Getting this error when trying to do `gatsby build`

> error "gatsby-transformer-rehype" threw an error while running the createSchemaCustomization lifecycle:
>  Cannot read property 'typeOrTypeDef' of undefined
>    TypeError: Cannot read property 'typeOrTypeDef' of undefined
>    
>    - create-schema-customization.js:13 type
> 

Gatsby CLI version: 2.11.15
Gatsby version: 2.22.15

After looking into the code it seems it's looking for plugin `jamify-source-ghost` which I don't have installed, which is why I think this error occurs. See code here:

```
const useTypeExists = (store, name) => type => {
  const types = store.getState().schemaCustomization.types;
  const plugin = types.find(node => node.plugin.name === name);
  const defs = plugin.typeOrTypeDef.definitions;
  const exists = defs.find(node => node.name.value === type);
  return exists !== undefined;
};

```

I would just comment this out normally, but I'm trying to do a CI/CD build and the build process is doing an npm install of all packages, including pulling in gatsby-transformer-rehype.
Could this be fixed?
Thank you!

**Comment by styxlab** [comment]
@glennsyang Thanks for using this project and for catching this bug!

Kind of interesting that you are using `gatsby-transformer-rehype` detached from `gatsby-starter-try-ghost`. I have fixed this bug in `gatsby-transformer-rehype@1.4.3`.

**Comment by glennsyang** [comment]
@styxlab wow! thanks for the quick response!
